packages/neptune-extension/src/common/components/input/ProjectInput.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import { backendClient } from 'common/api/backend-client';
+import { ProjectWithRoleDTO } from 'generated/backend-client/src/models';
+
+interface ProjectInputProps {
+  value: string
+  disabled: boolean
```

Comment:
I dont think disabled should be a mandatory prop. If dont pass disabled I would expect it to be equivalent to disabled = false

Reply:
This is just my small understanding of typescript.